README.md

```diff
@@ -682,7 +682,5 @@ If you need to install the agent through Ansible but don't want to specify an AP
 [18]: https://docs.datadoghq.com/security/cspm/setup/?tab=docker
 [19]: https://github.com/DataDog/integrations-core
 [20]: https://github.com/DataDog/integrations-extras
-[21]: https://github.com/DataDog/ansible-datadog/tree/nschweitzer/readme#integrations
-[22]: https://github.com/DataDog/ansible-datadog/tree/nschweitzer/readme#integrations-installation
 [23]: https://docs.datadoghq.com/tracing/trace_collection/library_injection_local/?tab=agentandappinseparatecontainers#configure-docker-injection
 [24]: https://docs.datadog.com/tracing/trace_collection/library_injection_local
```

Comment:
I can change these numbers to `[21]` and `[22]` respectively if you'd like to avoid having empty slots in the numbering here. I first opted to do a smaller changeset to make this PR easier to review but let me know if you'd like me to change these as well ☺️